docs/articles_en/openvino_workflow/running_inference_with_openvino/stateful_models_intro.rst

```diff
@@ -0,0 +1,121 @@
+.. {#openvino_docs_OV_UG_stateful_models_intro}
+
+Stateful models and State API
+==============================
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    openvino_docs_OV_UG_ways_to_get_stateful_model
+
+What is Stateful Model?
+#######################
+
+Stateful model is a model which stores the specified data between and during model inferences in internal memory space usually called State or Variable.
+In contrast to usual **stateless** model, which purify all the calculated data after each inference call, **stateful**
+model preserve tensors saved in States.
+
+OpenVINO Stateful Model Representation
+######################################
+
+OpenVINO contains ReadValue/Assign operations to make a model Stateful.
+Each pair of ReadValue/Assign operates with State, known also as Variable,
+which is an internal memory buffer to store tensor data during and between model inference calls.
+ReadValue reads data tensor from State and returns it as output, Assign accepts data tensor as input and writes data to State
+to save data for the next inference call.
```

Comment:
Move the first paragraph to a more low-level explanation below. Rename this section to Benefits of Stateful Models and list those 3 items below without mentioning ReadValue and Assign ops. Introduce low-level stuff gradually later. The beginning of the article shouldn't touch op-level specifics, first explain purpose, benefits and ways to obtain stateful models, then move to API specifics and inherent limitations that helps runtime to unlock that benefits that were promised in the beginning in comparison to regular Parameter/Results.